invenio_records_resources/services/files/service.py

```diff
@@ -103,10 +103,6 @@ def update_files_options(self, id_, identity, data):
         action = self.config.permission_action_prefix + "update_files"
         self.require_permission(identity, action, record=record)
 
-        # TODO: Maybe there's a better programmatic API to apply these?
```

Comment:
Eventually maybe? I favored starting small and targeted to the issue at hand. Because I was already introducing a lot of changes, I went on the conservative side (these options might also have large ramifications). I made an issue to transition them too: https://github.com/inveniosoftware/invenio-records-resources/issues/230